.yarnrc.yml

```diff
@@ -17,13 +17,23 @@ yarnPath: .yarn/releases/yarn-3.3.0.cjs
 
 packageExtensions:
   "@craco/craco@*":
+    dependencies:
+      postcss: "^8.1.0"
+      eslint: "^7.0.0 || ^8.0.0"
```

Comment:
**issue** according to [these logs](https://github.com/truvity/webapp-playground/pull/2#discussion_r1415622144) from yarn, no child packages requested `eslint` from `@craco/craco`. Let's not require more than we need.

**suggestion** 
```suggestion
```